ionic/src/app/api/device.service.ts

```diff
@@ -254,14 +250,17 @@ export class DeviceService {
                     } else {
                         this.notifyService.alertUser("unable to reach device. device not online");
                     }
-                } else if (res.type == "device_set_name") {
+                } else if (res.type == "device_set_name_reply") {
                     if (res.found) {
-                        this.notifyService.alertUser("device name recived")
+                        this.notifyService.alertUser("opertaion send to device")
                     }
                     else {
-                        this.notifyService.alertUser("unable to reach device name");
+                        this.notifyService.alertUser("unable to reach device.device is not online");
                     }
                 }
+                else if (res.type === "device_set_name_notify") {
+                    this.notifyService.alertUser("device name recieved")
```

Comment:
where are you updating device name in local storage?
it should be done here

Reply:
i upadte device name in home.ts file in local storage that's why i remove from here